huaweicloud/endpoints.go

```diff
@@ -33,22 +33,112 @@ var allServiceCatalog = map[string]ServiceCatalog{
 		Scope:            "global",
 		WithOutProjectID: true,
 	},
-
+	// ******* client for Compute start *******
 	"ecs": ServiceCatalog{
 		Name:    "ecs",
 		Version: "v1",
 	},
-	"compute": ServiceCatalog{
+	"computeV11": ServiceCatalog{
+		Name:    "ecs",
+		Version: "v1.1",
+	},
+	"computeV2": ServiceCatalog{
 		Name:    "ecs",
 		Version: "v2.1",
 	},
+	"autoscalingV1": ServiceCatalog{
+		Name:    "as",
+		Version: "autoscaling-api/v1",
+	},
+	"imageV2": ServiceCatalog{
+		Name:             "ims",
+		Version:          "v2",
+		WithOutProjectID: true,
+	},
+	"cceV3": ServiceCatalog{
+		Name:    "cce",
+		Version: "api/v3/projects",
+	},
+	"cceAddonV3": ServiceCatalog{
+		Name:             "cce",
+		Version:          "api/v3",
+		WithOutProjectID: true,
+	},
+	"cciV1": ServiceCatalog{
+		Name:             "cci",
+		Version:          "apis/networking.cci.io/v1beta1",
+		WithOutProjectID: true,
+	},
+	"FgsV2": ServiceCatalog{
+		Name:    "functiongraph",
+		Version: "v2",
+	},
+	// ******* client for Compute end  *******//
+
+	// ******* client for storage start ******//
+	"blockStorageV2": ServiceCatalog{
+		Name:    "evs",
+		Version: "v2",
+	},
+	"blockStorageV3": ServiceCatalog{
+		Name:    "evs",
+		Version: "v3",
+	},
+	"evsV2": ServiceCatalog{
```

Comment:
can we merge it with `blockStorageV2`?

Reply:
I will merge the same kind of client in next patch, may take a moment.